### PR TITLE
Change ALLOWED_HOSTS to DOMAIN

### LINKED ISSUE #59: Explicitly add domain to environment
The domain (example.com) is added to the environment in the `ALLOWED_HOSTS` key, but is more generally useful in other places, so I'd prefer to name it `DOMAIN`. See caktus/django-project-template#198

### PR DESCRIPTION
**Pull request by vkurup**
Closes #59

If accepted, the release notes will explain that users should grep for
instances of places that ALLOWED_HOSTS is set in their environment and change
it to DOMAIN instead.

### CODE REVIEW

**Comment by dpoirier** [comment]
Maybe go ahead and draft some text in CHANGES.rst?  We could title the section "Next release" or "Unreleased" or something like that.

**Comment by vkurup** [comment]
Good idea. Updated.

File: CHANGES.rst [hunk 1]
```diff
@@ -2,12 +2,29 @@ Margarita
...
+
+    ALLOWED_HOSTS = [os.environ['DOMAIN']]
+
 v 1.0.8 on Sep 4, 2015
 ----------------------
 
 * Add state to deploy elasticsearch (#72)
 * Note that New Relic high security shouldn't be enabled unless
   the account has it turned on. (#71)
+>>>>>>> develop
```
Comment (mlavin) [inline CHANGES.rst]: Left over merge conflict.

**Comment by mlavin** [comment]
It looks like this is blocking https://github.com/caktus/django-project-template/pull/200. What's left merge this? It looks good to me.

**Comment by vkurup** [comment]
Cool, thanks for the reviews!

**Pull request merged**